lib/mollie/order/line.rb

```diff
@@ -0,0 +1,55 @@
+module Mollie
+  class Order
+    class Line < Base
+      attr_accessor :id, :order_id, :type, :name, :status, :is_cancelable,
+                    :quantity, :quantity_shipped, :amount_shipped,
+                    :quantity_refunded, :amount_refunded, :quantity_canceled,
+                    :amount_canceled, :unit_price, :discount_amount,
+                    :total_amount, :vat_rate, :vat_amount, :sku, :image_url,
+                    :product_url, :created_at
+
+      def cancelable?
+        is_cancelable == true
+      end
+
+      def amount_shipped=(amount)
+        @amount_shipped = Mollie::Amount.new(amount)
+      end
+
+      def amount_refunded=(amount)
+        @amount_refunded = Mollie::Amount.new(amount)
+      end
+
+      def amount_canceled=(amount)
+        @amount_canceled = Mollie::Amount.new(amount)
+      end
+
+      def unit_price=(amount)
+        @unit_price = Mollie::Amount.new(amount)
+      end
+
+      def discount_amount=(amount)
+        @discount_amount = Mollie::Amount.new(amount)
+      end
+
+      def total_amount=(amount)
+        @total_amount = Mollie::Amount.new(amount)
+      end
+
```

Comment:
Can you add helper methods like:

```
line.shippable?
line.refundable?
line.cancelable?
```

They should basically check whether `quantity_shippable` / `quantity_refundable` / `quantity_cancelable` is > 0.

Reply:
👍 

For `cancelable?` we can continue to use `is_cancelable`, right?